### PR TITLE
Add Speakerdeck column to Speaker

### PR DESCRIPTION
**Pull request by marcoroth**
This pull request adds a `speakerdeck` column to the `speakers` table. Maybe this allows us in the future to automate the matching of `slides_url` to decks on Speakerdeck.

| Profile Sidebar | Suggestion |
| --- | --- |
| CleanShot 2024-10-20 at 19 52 48 | CleanShot 2024-10-20 at 19 52 53 |

### CODE REVIEW

File: db/migrate/20241020174649_add_speakerdeck_to_speaker.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddSpeakerdeckToSpeaker < ActiveRecord::Migration[8.0]
+  def change
+    add_column :speakers, :speakerdeck, :string
```
Comment (adrienpoly) [inline db/migrate/20241020174649_add_speakerdeck_to_speaker.rb:3]: I know it it controversial but I prefer for most cases to default string to "" with null: false

**Comment by adrienpoly** [comment]
great idea

File: db/migrate/20241020174649_add_speakerdeck_to_speaker.rb (hunk 1 above)
Comment (marcoroth) [reply db/migrate/20241020174649_add_speakerdeck_to_speaker.rb]: 👍🏼

**Comment by marcoroth** [comment]
Updated 🙌🏼

**Pull request merged**